website/content/docs/operators/observability/prometheus.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Observability with Prometheus
+---
+
+> The Prometheus metrics sink for Heron is available only if you're using [Kubernetes](../../deployment/schedulers/kubernetes) as your Heron scheduler.
+
+You can integrate Heron and the Heron UI with [Prometheus](https://prometheus.io/) to monitor and gather runtime metrics exported by Heron topologies.
+
+## Exporting topology metrics from Heron to Prometheus
+
+Heron supports custom metric exporters from the Metrics Manager. You can either build your own Prometheus metrics sink or use the [provided Prometheus sink](/docs/contributors/custom-metrics-sink/).
+
+To set up your Heron cluster to export to Prometheus, you need to make two changes to the `metrics_sinks.yaml` configuration file:
+
+* Add `prometheus-sink` to the `sinks` list (in addition to `file-sink`, `tmaster-sink`, and `metricscache-sink`).
+* Add a `prometheus-sink` map to the file that sets values for the [parameters](#prometheus-parameters) listed below
+
+### Prometheus parameters
+
+Parameter | Description | Default
+:---------|:------------|:-------
+`class` | The Java class used to control Prometheus sink behavior | [`com.twitter.heron.metricsmgr.sink.PrometheusSink`](/api/com/twitter/heron/metricsmgr/sink/PrometheusSink.html)
+`port` | The port on which Prometheus will scrape for metrics. **Note**: You must supply a `port` *or* `port-file` config. | 8080
+`port-file` | The path to a text file that contains the port number to be used by Prometheus for metrics scraping. **Note**: You must supply a `port` *or* `port-file` config. | `metrics.port`
+`path` | The Prometheus path on which to publish metrics | `/metrics`
+`flat-metrics` | Whether to publish a flat `name: value` JSON map | `true`
```

Comment:
We should remove this because the prometheus sink will always flatten the metrics.

Reply:
Got it